azext_edge/edge/_help.py

```diff
@@ -1935,7 +1935,10 @@ def load_iotops_help():
         type: command
         short-summary: Assign a user-assigned managed identity with the instance.
         long-summary: |
-            This operation includes federation of the identity.
+            This operation includes federation of the identity for the applicable purpose.
+
+            When --usage 'schema' is input, by default, a role assignment of the identity against the
```

Comment:
when --usage 'schema' is used

Reply:
That's what it was in the last commit but double "use" was a nit to avoid.